src/levels/remote/pushArgs.js

```diff
@@ -189,7 +189,7 @@ exports.level = {
             "markdowns": [
               "## Parámetros de push",
               "",
-              "¡Genial! Ahora que sabés acerca de las ramas que trackean remotos podemos empezar a develar algo del misterio detrás de git push, fetch y pull. Vamos a atacar de a un comando a la vez, pero los conceptos entre ellos son muy similares.",
+              "¡Genial! Ahora que sabés acerca de las ramas que trackean remotos podemos empezar a desvelar algo del misterio detrás de git push, fetch y pull. Vamos a atacar de a un comando a la vez, pero los conceptos entre ellos son muy similares.",
```

Comment:
`develar` and `desvelar` seem to mean the same.

Reply:
You are right!
https://dle.rae.es/?w=develar